### DISCUSSION TITLE
What if we need extra code to be written in onCreate() generated method ?

### DISCUSSION THREAD

**Discussion by IhabSaeed** [shinyorg/shiny#749] [Q&A] [answered]
Can we override onStart() to make that ?

**Comment by aritchie** [comment, marked as answer]
Yes.  The code gen skips methods that are already implemented.  In VS for windows, if you click on your android/ios project and break down "References > Analyzers > Shiny.Generators", you will be able to see all of the code that Shiny is generating.

I have also added a sample here: https://github.com/shinyorg/samples/tree/main/NoCodeGen

**Reply by IhabSaeed** [reply]
Thank you very much Allan
